controllers/trackController.js

```diff
@@ -1,131 +1,87 @@
-            }
-        });        
+};
+
+exports.trackAddAction = (req, res) => {
+    Track(req.body).save(function(err,data){
+        if(err) throw err;  
+        if(req.query.username!=null) {          
+            res.render("track-add",{tracks:data, title:"Track", username:req.query.username});
+        }
+        else {
+            res.render("track-add",{tracks:data, title:"Track", username:""});
+        }
+    });       
+};
+
+exports.trackDelete = (req, res) => {
+    Track.find({_id: req.params._id}).deleteOne(function(err,data) {                  
+        if(err) throw err;
+        if(req.query.username!=null) {
+            res.render("track-add",{tracks: data,title: "Add Track", username:req.query.username});
+        }
+        else {
+            res.render("track-add",{tracks: data,title: "Add Track", username:""});
+        }                  
     });
-    app.get('/cart',function(req,res){
-        CartItem.find({username: req.query.username},function(err,data){
-            if(err) throw err;
-            if(req.query.username!=null)
-            {
-                //console.log(data);
-                res.render('cart',{cartitems:data, title:'Shopping Cart', username:req.query.username});
-            }
-            else
-            {
-                res.render('cart',{cartitems:data, title:'Shopping Cart', username:''});
-            }               
-        });
+};
+
+exports.cartGet = (req, res) => {
+    CartItem.find({username: req.query.username},function(err,data) {
+        if(err) throw err;
+        if(req.query.username!=null) {
+            res.render("cart",{cartitems:data, title:"Shopping Cart", username:req.query.username});
+        }
+        else {
+            res.render("cart",{cartitems:data, title:"Shopping Cart", username:""});
+        }               
     });
-    app.delete('/cart/:_id',function(req,res){        
-        CartItem.find({_id: req.params._id})
-              .remove(function(err,data){                  
-                  if(err) throw err;
-                  if(req.query.username!=null)
-                    {
-                        res.render('cart',{cartitems:data, title:'Shopping Cart', username:req.query.username});
-                    }
-                    else
-                    {
-                        res.render('cart',{cartitems:data, title:'Shopping Cart', username:''});
-                    }              
-         });
-     });
-};
\ No newline at end of file
+};
+
+exports.cartAddItem = (req, res) => {
+    CartItem(req.body).save(function(err,data) {
+        if(err) throw err;   
+        if(req.query.username!=null) {
+            res.render("cart",{cartitems:data, title:"Shopping Cart", username:req.query.username});
+        }
+        else {
+            res.render("cart",{cartitems:data, title:"Shopping Cart", username:""});
+        }
+    });       
+};
+
+exports.cartDeleteItem = (req, res) => {

```

Comment:
try to be consistent with your exports :)
as a personal preference i usually export everything i need to on the last line, it makes everything easier to read :)